frontend/src/app/modules/lessons/lessons-page/lessons-page.component.html

```diff
@@ -7,12 +7,7 @@ <h2 class="date">{{ todayDate }}</h2>
 
     <div class="lessons-section" *ngIf="lessons">
         <div class="lessons-column">
-            <div class="lesson" *ngFor="let lesson of lessonsColumn1">
```

Comment:
this thing is mandatory
![image](https://user-images.githubusercontent.com/71650121/229549607-5414460d-55f1-485c-bc4c-028bb0139618.png)

Reply:
Return 2 columns, it's the only way to display cards with different height without white space

Reply:
ok